libraries/p5.simpleAR.yaml

```diff
@@ -0,0 +1,10 @@
+name: p5.simpleAR
+description: A simple JavaScript library that easily converts existing p5.js sketches to AR.
+category: 3d
+sourceUrl: https://github.com/tetunori/p5.simpleAR
+featuredImage:
+  url: p5.simpleAR.png
+  altText: A logo and description of 'p5.simpleAR' library are placed in the center.
+  name: Tetsunori Nakayama
```

Comment:
Looks like the `author:` key in this and in `p5.toio.yaml` get taken out, could I get you to add those back? Thanks!

Reply:
@davepagurek Thanks for your checking!! 
I seemed to delete `author` key. I will add them later!

Reply:
@davepagurek Since I have added `author` keys, could you kindly check them again?

Reply:
Looks good, thanks!